composer.json

```diff
@@ -28,7 +37,11 @@
     "prefer-stable": true,
     "scripts": {
         "post-install-cmd": "sh ./scripts/composer/post-install.sh",
-        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json"
+        "docker-swagger-generate": "docker-compose run swagger generate -i http://service:3000/explorer/swagger.json -l php -o /var/usr/client -c /var/usr/swagger/config.json",
```

Comment:
We cannot use Swagger-PHP. We use Swagger for generation of [webservice client code](https://github.com/DBCDK/biblo-admin/tree/develop/lib/client) from a Swagger specification. Swagger-PHP generates a Swagger specification from your webservice implementation.

My initial plan was not to generate client code as a part of the pipeline. The developer should do that as a part of the development process. If the specification changes and the generated code is updated during deployment then we may end up in a situation where the surrounding code calls a method that no longer exists. Such situations should be caught much earlier.